### PR TITLE
Issue #322: modified Getting Started section for clarity

### PR DESCRIPTION
**Pull request by shannonxtreme**
# Description

## install.rst

 - Added spacing before conda links
 - Updated the link formatting for the WSL URL to be compliant with rst
 - Removed unnecessary words from Install the package

## quick_start.md

- Updated link to Command Line Tools in intro
 - Added colons after "where" to introduct lists
 - Added <filepath> variables instead of copy-pasted paths in lists

## Type of change

- [x] This change is a documentation update

### CODE REVIEW

File: docs/source/getting_started/quick_start.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Quick Start
 
-Following is a simplified demo using example dataset to train a registration network.
-More details and other options can be found in the [command line tools](doc_command.md).
+This is a simple demo using an example dataset to train a registration network.
+More details and other options can be found in [Command line tools](cli.md).
```
Comment (mathpluscode) [inline docs/source/getting_started/quick_start.md:4]: thx for fixing the wrong link, the `cli.md` might need to be replaced by `../docs/cli.md`

I didn't have the chance to add the docs for using readthedocs yet, but i belive you can test locally with `make clean html` under `docs/`

**Review by mathpluscode — CHANGES_REQUESTED** [review]
please correct the link to cli.md ;) thx

**Comment by shannonxtreme** [comment]
> 
> 
> please correct the link to cli.md ;) thx

Will do! Side note, not sure how to test locally. Is there a guide somewhere? Thanks!

**Comment by mathpluscode** [comment]
> > please correct the link to cli.md ;) thx
> 
> Will do! Side note, not sure how to test locally. Is there a guide somewhere? Thanks!

Hi, I just added a page regarding the docs here https://deepreg.readthedocs.io/en/325-improve-contributing-pages/contributing/docs.html It will be merged to master later.

Also, there's a new page about setting up your loader dev env here https://deepreg.readthedocs.io/en/325-improve-contributing-pages/contributing/setup.html.

Please feel free to give us feedback on these pages and other docs. Thanks!

**Review by mathpluscode — APPROVED** [review]

**Comment by shannonxtreme** [comment]
> Side note: your last commit message was "Update cli.md link with correct dir structure", please include the issue number next time as suggested here https://deepreg.readthedocs.io/en/latest/contributing/commit.html ;) thx for understanding!

I'm so sorry! It's automatic sometimes for me. I'll pay more attention! Thanks for the links :)

**Pull request merged**